tools/schemacode/bidsschematools/render/text.py

```diff
@@ -209,6 +209,7 @@ def value_key_table(namespace):
 
 
 def make_filename_template(
+    file_type,
```

Comment:
Should this be "dataset_type"? "file_type" makes me think "directory" vs. "regular" based on the other changes in this PR.

Reply:
The idea was that the rules are for files more than for datasets, but that's a fair point. Happy to change to `dataset_type` unless someone has a better/clearer suggestion.